xbmc/pvr/epg/Epg.cpp

```diff
@@ -552,3 +553,17 @@ void CPVREpg::RemovedFromContainer()
 {
   m_events.Publish(PVREvent::EpgDeleted);
 }
+
+int CPVREpg::CleanupCachedImages(const std::shared_ptr<CPVREpgDatabase>& database)
+{
+  std::vector<std::string> urlsToCheck;
+
+  const std::vector<std::string> paths = database->GetAllIconPaths(EpgID());
```

Comment:
What's the use of `paths` and the following loop? Wouldn't assigning directly to `urlsToCheck` do the same thing?

Reply:
Good catch. Thanks. Must be a leftover from an earlier iteration of this PR.